localedata/locales/ab_GE

```diff
@@ -50,58 +50,52 @@ copy "ka_GE"
 END LC_COLLATE
 
 LC_TIME
-% Ажьырныҳəа, Жəабран, Хəажəкыра, Мшаԥы, Лаҵара, Рашəара, Ԥхынгəы, Нанҳəа, Цəыббра, Жьҭаара, Абҵара, Ԥхынҷкәын
-mon          "<U0410><U0436><U044C><U044B><U0440><U043D><U044B><U04B3><U04d9><U0430>";/
```

Comment:
looks like the files are utf-8 now but there are still some escape sequences in some files